tests/unit/api/v1/questions/test_update_question.py

```diff
@@ -111,3 +111,32 @@ async def test_update_question_with_more_visible_options_than_allowed(
         assert response.json() == {
             "detail": "the value for 'visible_options' must be less or equal to the number of items in 'options' (3)"
         }
+
+    async def test_update_span_question_with_unsupported_allow_overlapping_value(
```

Comment:
```suggestion
    async def test_update_span_question_disabling_allow_overlapping(
```

Reply:
Maybe another one checking that it's possible to enable `allow_overlapping`?

Something like `test_update_span_question_enabling_allow_overlapping`.